codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Headers.java

```diff
@@ -146,6 +146,12 @@ public CharSequence status() {
     }
 
     @Override
+    public Http2Headers clear() {
```

Comment:
I wonder if we do an extra pr for this as this is not really related but a bug that you fixed. If we do an extra pr it is easier to keep track of changes for the user and see if an issue is fixed for them.